### PR TITLE
Add an experimental tools directory, including a PoC Datalab manager.

### PR DESCRIPTION
**Pull request by ojarjur**

### CODE REVIEW

File: tools/experimental/hosted-service/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Managed Datalab Service Example
+
+This directory shows how you can run you own service for automatically
```
Comment (yebrahim) [inline tools/experimental/hosted-service/README.md]: typo: your* own

File: tools/experimental/hosted-service/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+```sh
+pushd ./
+cd /tmp
+git clone https://github.com/google/inverting-proxy
+cd inverting-proxy
+make deploy
+popd
+```
+
+Next, deploy the the `datalabstatic` App Engine service to your project.
```
Comment (yebrahim) [inline tools/experimental/hosted-service/README.md]: typo: the the

File: tools/experimental/hosted-service/README.md (hunk 1 above)
Comment (ojarjur) [reply tools/experimental/hosted-service/README.md]: Done

File: tools/experimental/hosted-service/README.md (hunk 2 above)
Comment (ojarjur) [reply tools/experimental/hosted-service/README.md]: Done.

**Pull request merged**